### PR TITLE
Make sure the entity name is used as type in the resource response

### PR DESCRIPTION
**Pull request by Marlinc**

### CODE REVIEW

File: src/Listener/JsonApiListener.php [hunk 1]
```diff
@@ -470,6 +470,7 @@ protected function _stripNonContainedAssociations($table, $entity)
      * @param string $entityName Name of the current entity
      * @param \Cake\ORM\AssociationCollection $associations AssciationCollection
      * @return array
+     * @internal
```
Comment (ADmad) [inline src/Listener/JsonApiListener.php]: Why?

Comment (ADmad) [inline src/Listener/JsonApiListener.php]: Ok

**Comment by bravo-kernel** [comment]
This looks good to me

**Comment by bravo-kernel** [comment]
Could this be merged please?

**Comment by bravo-kernel** [comment]
Thanks :thumb:

**Comment by bravo-kernel** [comment]
Awesome job mr. @Marlinc, my plugin entity is now detected where it failed hard before. 

Much obliged 👏

**Pull request merged**